source/plugin/text/text.rst

```diff
@@ -40,6 +40,20 @@ The text builder interface allows for the creation of formatted text in a "build
     Read this `Wikipedia article <https://en.wikipedia.org/wiki/Builder_pattern>`__ for help understanding the purpose
     of the builder pattern in software design.
 
+General Procedure
+~~~~~~~~~~~~~~~~~
+
+1. Start with <code>Text.builder()</code>.
```

Comment:
Replace all `<code>Text</code>` blocks with ` ``Text`` ` blocks.

Reply:
As an alternative you could also use javadoc links : ``:javadoc:`Text#builder()` ``